pkg/api/types.go

```diff
@@ -1,8 +1,9 @@
 package api
 
 import (
-	docker "github.com/fsouza/go-dockerclient"
 	"time"
+
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
I really don't know how it works. For me it leaves them as they were before or orders them this way if I put a line between.. This creates unnecessary merge problems when two PRs are reordering those (already happened here).